editor/src/components/navigator/left-pane/github-pane/index.tsx

```diff
@@ -469,6 +481,24 @@ export const GithubPane = React.memo(() => {
             </>,
           )}
           {loadBranchesUI}
+          {when(
+            currentBranch != null,
+            <UIGridRow
+              padded
+              variant='<-------------1fr------------->'
+              style={{ margin: '10px 0' }}
+            >
+              <Button
+                spotlight
+                highlight
+                style={{ backgroundColor: '#fee', border: '1px solid #f00' }}
```

Comment:
question / suggestion: if this is a non-destructive change ("just updates project settings"), I wouldn't use scaryRed here but a subdued button style.

If it _is_ a destructive change (you will lose project contents in some way), use the `colorTheme` (pretty sure we have a warningColor in there somewhere). I'd still make this a subdued button (i.e. "red text only" unless you hover) since it's a pretty infrequent action.

Reply:
sgtm, removed the scary red color for a default button → https://github.com/concrete-utopia/utopia/pull/2794/commits/a13651986e0e8a14311213b22a4cf7447e29b0fb